auditbeat/docs/configuring-howto.asciidoc

```diff
@@ -36,6 +36,7 @@ The following topics describe how to configure {beatname_uc}:
 * <<yaml-tips>>
 * <<regexp-support>>
 * <<{beatname_lc}-reference-yml>>
+* <<configuring-monitoring>>
```

Comment:
I'd rather that this topic come after `<<configuration-logging>>` for all the Beats. If you are pressed for time, I can do the work after you merge this PR. I think it would be good to get this in quickly.